tests/integration-tests/tests/api/test_api_client.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+
+import pytest
+import base64
+from assertpy import assert_that
+
+from tests.common.utils import generate_random_string
+
+# Client Interface
+from pcluster_client.model.create_cluster_request_content import CreateClusterRequestContent
+
+@pytest.mark.regions(["us-east-2"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.schedulers(["slurm"])
+@pytest.mark.oss(["alinux2"])
```

Comment:
feel free to remove these markers that are not required with the new test config approach. we will have to remove them from every test function sooner or later